grommet/grommet
# Select: fixed scroll reset when clicking multiple select options

#### What does this PR do?
Fixes DropContainer to not reset top position. Whenever we do that the browser will scroll to the top automatically and we don't want that to happen.

#### Where should the reviewer start?
DropContainer
#### What testing has been done on this PR?

#### How should this be manually tested?
open select Custom Search example and make sure that the scroll does not change when selecting items
#### Any background context you want to provide?

#### What are the relevant issues?
Fixes https://github.com/grommet/grommet/issues/2415

#### Screenshots (if appropriate)

#### Do the grommet docs need to be updated?
no
#### Should this PR be mentioned in the release notes?
yes
#### Is this change backwards compatible or is it a breaking change?
backwards
